translations/kubectl/pt_BR/LC_MESSAGES/k8s.po

```diff
@@ -0,0 +1,3319 @@
+
+#: pkg/kubectl/cmd/options.go:38
+msgid "Print the list of flags inherited by all commands"
+msgstr "Mostra a list de opções herdadas por todos os comandos"
+
+#: pkg/kubectl/cmd/logs.go:93
+msgid "Print the logs for a container in a pod"
+msgstr "Mostra os logs de um container em um pod"
+
+#: pkg/kubectl/cmd/replace.go:71
+msgid "Replace a resource by filename or stdin"
+msgstr "Substitui um recurso por um nome de arquivo ou stdin"
+
+#: pkg/kubectl/cmd/rollout/rollout_resume.go:72
+msgid "Resume a paused resource"
+msgstr "Retoma um recurso pausado"
+
+#: pkg/kubectl/cmd/create_rolebinding.go:57
+msgid "Role this RoleBinding should reference"
+msgstr "Role que a RoleBinding deve referenciar"
+
+#: pkg/kubectl/cmd/run.go:97
+msgid "Run a particular image on the cluster"
+msgstr "Executa uma imagem específica no cluster"
+
+#: pkg/kubectl/cmd/proxy.go:69
+msgid "Run a proxy to the Kubernetes API server"
+msgstr "Executa um proxy para o servidor de API do Kubernetes"
+
+#: pkg/kubectl/cmd/create_secret.go:161
+msgid "Server location for Docker registry"
+msgstr "Localização do servidor para o registro do Docker"
+
+#: pkg/kubectl/cmd/scale.go:71
+msgid ""
+"Set a new size for a Deployment, ReplicaSet, Replication Controller, or Job"
+msgstr ""
+"Define um novo tamanho para um Deployment, ReplicaSet, Replication "
+"Controller, ou Job"
+
+#: pkg/kubectl/cmd/set/set.go:38
+msgid "Set specific features on objects"
+msgstr "Define funcionalidades específicas em objetos"
+
+#: pkg/kubectl/cmd/apply_set_last_applied.go:83
+msgid ""
+"Set the last-applied-configuration annotation on a live object to match the "
+"contents of a file."
+msgstr ""
+"Define a anotação last-applied-configuration em um live object para "
+"corresponder ao conteúdo do arquivo."
+
+#: pkg/kubectl/cmd/set/set_selector.go:82
+msgid "Set the selector on a resource"
+msgstr "Define um seletor em um recurso"
+
+#: pkg/kubectl/cmd/config/create_cluster.go:68
+msgid "Sets a cluster entry in kubeconfig"
+msgstr "Define um cluster no arquivo kubeconfig"
+
+#: pkg/kubectl/cmd/config/create_context.go:58
+msgid "Sets a context entry in kubeconfig"
+msgstr "Define um context no arquivo kubeconfig"
+
+#: pkg/kubectl/cmd/config/create_authinfo.go:104
+msgid "Sets a user entry in kubeconfig"
+msgstr "Define um usuário no arquivo kubeconfig"
+
+#: pkg/kubectl/cmd/config/set.go:60
+msgid "Sets an individual value in a kubeconfig file"
+msgstr "Define um valor individual no arquivo kubeconfig"
+
+#: pkg/kubectl/cmd/config/use_context.go:49
+msgid "Sets the current-context in a kubeconfig file"
+msgstr "Define o current-context no arquivo kubeconfig"
+
+#: pkg/kubectl/cmd/describe.go:86
+msgid "Show details of a specific resource or group of resources"
+msgstr "Mostra os detalhes de um recurso específico ou de um grupo de recursos"
+
+#: pkg/kubectl/cmd/rollout/rollout_status.go:58
+msgid "Show the status of the rollout"
+msgstr "Mostra o status de um rollout"
```

Comment:
The rollout word kind of bothers me (the lack of translation), but honestly I couldn't find something better, so just leaving here for a future us when we get some better idea